### PR TITLE
[feat] added mmcif get_model method

### PR DESCRIPTION
**Pull request by kierandidi**
### Description

Added `get_model` and `get_models` methods to `PandasMmcif` class, similar to `PandasMmtf`.

### Pull Request Checklist

- [ ] Added a note about the modification or contribution to the `./docs/sources/CHANGELOG.md` file (if applicable)
- [ ] Added appropriate unit test functions in the `./biopandas/*/tests` directories (if applicable)
- [ ] Modify documentation in the corresponding Jupyter Notebook under `biopandas/docs/sources/` (if applicable)
- [ ] Ran `PYTHONPATH='.' pytest ./biopandas -sv` and make sure that all unit tests pass (for small modifications, it might be sufficient to only run the specific test file, e.g., `PYTHONPATH='.' pytest ./biopandas/classifier/tests/test_stacking_cv_classifier.py -sv`)
- [ ] Checked for style issues by running `flake8 ./biopandas`

### CODE REVIEW

File: biopandas/mmcif/pandas_mmcif.py [hunk 1]
```diff
@@ -68,6 +69,58 @@ def read_mmcif(self, path):
...
+            An integer representing the model index to subset to.
+
+        Returns
+        ---------
+        pandas_pdb.PandasPdb : A new PandasMMcif object containing the
+            structure subsetted to the given model.
+        """
+
+        df = copy.deepcopy(self)
+        breakpoint()
```
Comment (a-r-j) [inline biopandas/mmcif/pandas_mmcif.py:89]: This should be removed :)

File: biopandas/mmcif/pandas_mmcif.py [hunk 2]
```diff
@@ -68,6 +69,58 @@ def read_mmcif(self, path):
...
+        model_index : int
+            An integer representing the model index to subset to.
+
+        Returns
+        ---------
+        pandas_pdb.PandasPdb : A new PandasMMcif object containing the
+            structure subsetted to the given model.
+        """
+
+        df = copy.deepcopy(self)
```
Comment (a-r-j) [inline biopandas/mmcif/pandas_mmcif.py:88]: Would rename from `df` since this is not a dataframe.

File: biopandas/mmcif/pandas_mmcif.py [hunk 3]
```diff
@@ -68,6 +69,58 @@ def read_mmcif(self, path):
...
+
+        Returns
+        ---------
+        pandas_pdb.PandasMmtf : A new PandasMmcif object
+            containing the structure subsetted to the given model.
+        """
+
+        df = copy.deepcopy(self)
+
+        if "ATOM" in df.df.keys():
```
Comment (a-r-j) [inline biopandas/mmcif/pandas_mmcif.py:115]: as above

Comment (kierandidi) [reply biopandas/mmcif/pandas_mmcif.py:115]: Took that pattern from the pandas_pdb and pandas_mmtf file, but can change all three of them

Comment (kierandidi) [reply biopandas/mmcif/pandas_mmcif.py]: named it `atomic_df` for now to adhere with patterns in the source code

Comment (a-r-j) [reply biopandas/mmcif/pandas_mmcif.py]: I wouldn't use `df`, since it's a `PandasMmcif` object. Maybe: `pmmcif`

Comment (kierandidi) [reply biopandas/mmcif/pandas_mmcif.py]: fair, tried to keep it generic so the method can be the same in the three different file objects. Maybe `pandas_object`? Or `pandas_structure`?

Comment (a-r-j) [reply biopandas/mmcif/pandas_mmcif.py]: Either is fine but I'd opt for `biopandas` over `pandas` to make this distinction clear.

**Pull request closed without merging**